<!DOCTYPE html>
<html lang="zh-CN,en,default">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<meta name="theme-color" content="#222">
<meta name="generator" content="Hexo 5.4.0">
  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon-next.png">
  <link rel="icon" type="image/png" sizes="32x32" href="/images/orange.png">
  <link rel="icon" type="image/png" sizes="16x16" href="/images/orange.png">
  <link rel="mask-icon" href="/images/logo.svg" color="#222">
  <meta name="google-site-verification" content="HMAT9TapFXG6PRtARF9Qc_fDKANMj-rP3F5FjHjr7kI">

<link rel="stylesheet" href="/css/main.css">

<link rel="stylesheet" href="//fonts.googleapis.com/css?family=Noto Serif SC:300,300italic,400,400italic,700,700italic|Roboto Mono:300,300italic,400,400italic,700,700italic&display=swap&subset=latin,latin-ext">
<link rel="stylesheet" href="/lib/font-awesome/css/font-awesome.min.css">
  <link rel="stylesheet" href="//cdn.jsdelivr.net/gh/fancyapps/fancybox@3/dist/jquery.fancybox.min.css">


<script id="hexo-configurations">
  var NexT = window.NexT || {};
  var CONFIG = {
    hostname: new URL('https://dana5haw.com').hostname,
    root: '/',
    scheme: 'Gemini',
    version: '7.7.0',
    exturl: false,
    sidebar: {"position":"left","display":"post","padding":18,"offset":12,"onmobile":false},
    copycode: {"enable":true,"show_result":true,"style":"mac"},
    back2top: {"enable":true,"sidebar":false,"scrollpercent":true},
    bookmark: {"enable":true,"color":"#222","save":"auto"},
    fancybox: true,
    mediumzoom: false,
    lazyload: false,
    pangu: false,
    comments: {"style":"tabs","active":null,"storage":true,"lazyload":false,"nav":null},
    algolia: {
      appID: '',
      apiKey: '',
      indexName: '',
      hits: {"per_page":10},
      labels: {"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}
    },
    localsearch: {"enable":true,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false},
    path: './public/search.xml',
    motion: {"enable":true,"async":false,"transition":{"post_block":"fadeIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}}
  };
</script>
<script data-ad-client="ca-pub-3938956891324222" async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>

<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-203752309-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-203752309-1');
</script>
  <meta name="description" content="前言鉴于对2019-nCov病毒的关注，最近得知印度科学家发表了一篇论文，论证该病毒被插入了HIV病毒的氨基酸序列，新闻报导很多。为了辨别这些新闻的真伪，在网上找了论文原版，自己尝试着翻译一下。">
<meta property="og:type" content="article">
<meta property="og:title" content="2019-nCov病毒">
<meta property="og:url" content="https://dana5haw.com/posts/ZH-CN/2019-nCov.html">
<meta property="og:site_name" content="Dana Shaw&#39;s blog">
<meta property="og:description" content="前言鉴于对2019-nCov病毒的关注，最近得知印度科学家发表了一篇论文，论证该病毒被插入了HIV病毒的氨基酸序列，新闻报导很多。为了辨别这些新闻的真伪，在网上找了论文原版，自己尝试着翻译一下。">
<meta property="og:locale" content="zh_CN">
<meta property="article:published_time" content="2020-02-02T01:55:26.000Z">
<meta property="article:modified_time" content="2021-08-06T03:36:22.395Z">
<meta property="article:author" content="Dana Shaw">
<meta property="article:tag" content="翻译">
<meta property="article:tag" content="2019-nCov病毒">
<meta name="twitter:card" content="summary">

<link rel="canonical" href="https://dana5haw.com/posts/ZH-CN/2019-nCov.html">


<script id="page-configurations">
  // https://hexo.io/docs/variables.html
  CONFIG.page = {
    sidebar: "",
    isHome: false,
    isPost: true
  };
</script>

  <title>2019-nCov病毒 | Dana Shaw's blog</title>
  
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-203752309-1"></script>
    <script>
      var host = window.location.host;
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      if (host.indexOf("127.0.0.1") == -1 && host.indexOf("localhost") == -1) {
        gtag('js', new Date());
        gtag('config', 'UA-203752309-1');
      }
    </script>






  <noscript>
  <style>
  .use-motion .brand,
  .use-motion .menu-item,
  .sidebar-inner,
  .use-motion .post-block,
  .use-motion .pagination,
  .use-motion .comments,
  .use-motion .post-header,
  .use-motion .post-body,
  .use-motion .collection-header { opacity: initial; }

  .use-motion .site-title,
  .use-motion .site-subtitle {
    opacity: initial;
    top: initial;
  }

  .use-motion .logo-line-before i { left: initial; }
  .use-motion .logo-line-after i { right: initial; }
  </style>
</noscript>

<link rel="alternate" href="/atom.xml" title="Dana Shaw's blog" type="application/atom+xml">
</head>

<body itemscope itemtype="http://schema.org/WebPage">
  <div class="container use-motion">
    <div class="headband"></div>

    <header class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-container">
  <div class="site-meta">

    <div>
      <a href="/" class="brand" rel="start">
        <span class="logo-line-before"><i></i></span>
        <span class="site-title">Dana Shaw's blog</span>
        <span class="logo-line-after"><i></i></span>
      </a>
    </div>
        <p class="site-subtitle">为了去往别处</p>
  </div>

  <div class="site-nav-toggle">
    <div class="toggle" aria-label="切换导航栏">
      <span class="toggle-line toggle-line-first"></span>
      <span class="toggle-line toggle-line-middle"></span>
      <span class="toggle-line toggle-line-last"></span>
    </div>
  </div>
</div>


<nav class="site-nav">
  
  <ul id="menu" class="menu">
        <li class="menu-item menu-item-home">

    <a href="/" rel="section"><i class="fa fa-fw fa-home"></i>首页</a>

  </li>
        <li class="menu-item menu-item-about">

    <a href="/about/" rel="section"><i class="fa fa-fw fa-user"></i>关于</a>

  </li>
        <li class="menu-item menu-item-tags">

    <a href="/tags/" rel="section"><i class="fa fa-fw fa-tags"></i>标签</a>

  </li>
        <li class="menu-item menu-item-categories">

    <a href="/categories/" rel="section"><i class="fa fa-fw fa-th"></i>分类</a>

  </li>
        <li class="menu-item menu-item-archives">

    <a href="/archives/" rel="section"><i class="fa fa-fw fa-archive"></i>归档</a>

  </li>
        <li class="menu-item menu-item-sitemap">

    <a href="/sitemap.xml" rel="section"><i class="fa fa-fw fa-sitemap"></i>站点地图</a>

  </li>
        <li class="menu-item menu-item-commonweal">

    <a href="/404/" rel="section"><i class="fa fa-fw fa-heartbeat"></i>公益 404</a>

  </li>
        <li class="menu-item menu-item-readnotes">

    <a href="/categories/%E8%AF%BB%E4%B9%A6%E7%AC%94%E8%AE%B0/" rel="section"><i class="fa fa-fw fa-book"></i>读书笔记</a>

  </li>
        <li class="menu-item menu-item-movienotes">

    <a href="/categories/%E8%A7%82%E5%BD%B1%E7%AC%94%E8%AE%B0/" rel="section"><i class="fa fa-fw fa-youtube"></i>观影笔记</a>

  </li>
      <li class="menu-item menu-item-search">
        <a role="button" class="popup-trigger"><i class="fa fa-search fa-fw"></i>搜索
        </a>
      </li>
  </ul>

</nav>
  <div class="site-search">
    <div class="popup search-popup">
    <div class="search-header">
  <span class="search-icon">
    <i class="fa fa-search"></i>
  </span>
  <div class="search-input-container">
    <input autocomplete="off" autocorrect="off" autocapitalize="none"
           placeholder="搜索..." spellcheck="false"
           type="text" id="search-input">
  </div>
  <span class="popup-btn-close">
    <i class="fa fa-times-circle"></i>
  </span>
</div>
<div id="search-result"></div>

</div>
<div class="search-pop-overlay"></div>

  </div>
</div>
    </header>

    
  <div class="back-to-top">
    <i class="fa fa-arrow-up"></i>
    <span>0%</span>
  </div>
  <a role="button" class="book-mark-link book-mark-link-fixed"></a>


    <main class="main">
      <div class="main-inner">
        <div class="content-wrap">
          

          <div class="content">
            

  <div class="posts-expand">
      
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block " lang="ZH-CN">
    <link itemprop="mainEntityOfPage" href="https://dana5haw.com/posts/ZH-CN/2019-nCov.html">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/./images/photo.jpg">
      <meta itemprop="name" content="Dana Shaw">
      <meta itemprop="description" content="连尘埃也想收藏">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="Dana Shaw's blog">
    </span>
      <header class="post-header">
        <h1 class="post-title" itemprop="name headline">
          2019-nCov病毒
        </h1>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="fa fa-calendar-o"></i>
              </span>
              <span class="post-meta-item-text">发表于</span>

              <time title="创建时间：2020-02-02 09:55:26" itemprop="dateCreated datePublished" datetime="2020-02-02T09:55:26+08:00">2020-02-02</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="fa fa-calendar-check-o"></i>
                </span>
                <span class="post-meta-item-text">更新于</span>
                <time title="修改时间：2021-08-06 11:36:22" itemprop="dateModified" datetime="2021-08-06T11:36:22+08:00">2021-08-06</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="fa fa-folder-o"></i>
              </span>
              <span class="post-meta-item-text">分类于</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/%E7%BF%BB%E8%AF%91/" itemprop="url" rel="index">
                    <span itemprop="name">翻译</span>
                  </a>
                </span>
            </span>

          
            <span class="post-meta-item" title="阅读次数" id="busuanzi_container_page_pv" style="display: none;">
              <span class="post-meta-item-icon">
                <i class="fa fa-eye"></i>
              </span>
              <span class="post-meta-item-text">阅读次数：</span>
              <span id="busuanzi_value_page_pv"></span>
            </span><br>
            <span class="post-meta-item" title="本文字数">
              <span class="post-meta-item-icon">
                <i class="fa fa-file-word-o"></i>
              </span>
                <span class="post-meta-item-text">本文字数：</span>
              <span>21k</span>
            </span>
            <span class="post-meta-item" title="阅读时长">
              <span class="post-meta-item-icon">
                <i class="fa fa-clock-o"></i>
              </span>
                <span class="post-meta-item-text">阅读时长 &asymp;</span>
              <span>19 分钟</span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
        <h2 id="前言"><a href="#前言" class="headerlink" title="前言"></a>前言</h2><p>鉴于对2019-nCov病毒的关注，最近得知印度科学家发表了一篇论文，论证该病毒被插入了HIV病毒的氨基酸序列，新闻报导很多。为了辨别这些新闻的真伪，在网上找了论文原版，自己尝试着翻译一下。</p>
<p><escape> <span id="more"></span></escape></p>
<h2 id="正文"><a href="#正文" class="headerlink" title="正文"></a>正文</h2><blockquote>
<p><strong>Uncanny similarity of unique inserts in the 2019-nCoV spike protein to HIV-1 gp120 and Gag</strong></p>
</blockquote>
<p><strong>2019新型冠状病毒棘突蛋白的特殊插入序列和HIV-1 gp120基因、Gag基因有神秘的相似性</strong></p>
<blockquote>
<p>Abstract:<br>We are currently witnessing a major epidemic caused by the 2019 novel coronavirus (2019-nCoV). The evolution of 2019-nCoV remains elusive. We found 4 insertions in the spike glycoprotein (S) which are unique to the 2019-nCoV and are not present in other coronaviruses. Importantly, amino acid residues in all the 4 inserts have identity or similarity to those in the HIV-1 gp120 or HIV-1 Gag. Interestingly, despite the inserts being discontinuous on the primary amino acid sequence, 3D-modelling of the 2019-nCoV suggests that they converge to constitute the receptor binding site. The finding of 4 unique inserts in the 2019-nCoV, all of which have identity /similarity to amino acid residues in key structural proteins of HIV-1 is unlikely to be fortuitous in nature. This work provides yet unknown insights on 2019-nCoV and sheds light on the evolution and pathogenicity of this virus with important implications for diagnosis of this virus.</p>
</blockquote>
<p>摘要：<br>我们目前正在见证着2019新型冠状病毒导致的一场严重疫情。病毒的进化仍然难以捉摸。我们发现这个病毒的棘突糖蛋白（S蛋白）有4个特殊的插入序列，其他的冠状病毒也没有这样的插入序列。重点是，这4个序列的氨基酸残基具有HIV-1 gp120基因、HIV-1 Gag基因的特征。有趣的是，尽管这些插入序列在一级氨基酸序列上是不连续的，2019-nCov的3D模型却显示，它们汇聚并构成了受体结合部位。2019-nCov的4个特殊插入序列具有和HIV-1主要结构蛋白相似的氨基酸残基这一发现，不可能是自然界偶发的。本论文提供了2019-nCov的未知见解，阐明这个病毒的进化和致病性，对分析病毒有重大意义。</p>
<blockquote>
<p>Introduction<br>Coronaviruses (CoV) are single-stranded positive-sense RNA viruses that infect animals and humans. These are classified into 4 genera based on their host specificity: Alphacoronavirus, Betacoronavirus, Deltacoronavirus and Gammacoronavirus (Snijder et al., 2006). There are seven known types of CoVs that includes 229E and NL63 (Genus Alphacoronavirus), OC43, HKU1, MERS and SARS (Genus Betacoronavirus). While 229E, NL63, OC43, and HKU1 commonly infect humans, the SARS and MERS outbreak in 2002 and 2012 respectively occurred when the virus crossed-over from animals to humans causing significant mortality (J. Chan et al., n.d.; J. F. W. Chan et al., 2015). In December 2019, another outbreak of coronavirus was reported from Wuhan, China that also transmitted from animals to humans. This new virus has been temporarily termed as 2019-novel Coronavirus (2019-nCoV) by the World Health Organization (WHO) (J. F.- W. Chan et al., 2020; Zhu et al., 2020). While there are several hypotheses about the origin of 2019-nCoV, the source of this ongoing outbreak remains elusive. </p>
</blockquote>
<p>介绍<br>冠状病毒（CoV）是单链正义RNA病毒，会感染动物和人类。基于宿主专一性，分为4类：α冠状病毒、β冠状病毒、Δ冠状病毒、γ冠状病毒。已知的7类冠状病毒包括：229E和NL63（α冠状病毒属）、OC43、HKU1、MERS和SARS（β冠状病毒属）。229E、NL63、OC43、HKU1通常感染人类，2002年和2012年分别爆发的SARS和MERS由动物传播到人类，造成巨大死亡。2019年12月，中国武汉报导的一起冠状病毒爆发，也是由动物传播到人类。这种新型病毒由世卫组织（WHO）暂命名为2019新型冠状病毒。尽管关于2019-nCov的来源有几种假设，这个病毒的起源仍然难以捉摸。</p>
<blockquote>
<p>The transmission patterns of 2019-nCoV is similar to patterns of transmission documented in the previous outbreaks including by bodily or aerosol contact with persons infected with the virus. Cases of mild to severe illness, and death from the infection have been reported from Wuhan. This outbreak has spread rapidly distant nations including France, Australia and USA among others. The number of cases within and outside China are increasing steeply. Our current understanding is limited to the virus genome sequences and modest epidemiological and clinical data. Comprehensive analysis of the available 2019- nCoV sequences may provide important clues that may help advance our current understanding to manage the ongoing outbreak.<br>The spike glycoprotein (S) of cornonavirus is cleaved into two subunits (S1 and S2). The S1 subunit helps in receptor binding and the S2 subunit facilitates membrane fusion (Bosch et al., 2003; Li, 2016). The spike glycoproteins of coronoviruses are important determinants of tissue tropism and host range. In addition the spike glycoproteins are critical targets for vaccine development (Du et al., 2013). For this reason, the spike proteins represent the most extensively studied among coronaviruses. We therefore sought to investigate the spike glycoprotein of the 2019-nCoV to understand its evolution, novel features sequence and structural features using computational tools. </p>
</blockquote>
<p>2019-nCov的传播模式类似于以往爆发的记录在案的病毒，通过身体、空气接触传播病毒。</p>
<blockquote>
<p>Methodology<br>Retrieval and alignment of nucleic acid and protein sequences<br>We retrieved all the available coronavirus sequences (n=55) from NCBI viral genome database<br>(<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/">https://www.ncbi.nlm.nih.gov/</a>) and we used the GISAID (Elbe &amp; Buckland-Merrett,<br>2017)[<a target="_blank" rel="noopener" href="https://www.gisaid.org/]">https://www.gisaid.org/]</a> to retrieve all available full-length sequences (n=28) of 2019-<br>nCoV as on 27 Jan 2020. Multiple sequence alignment of all coronavirus genomes was performed<br>by using MUSCLE software (Edgar, 2004) based on neighbour joining method. Out of 55<br>coronavirus genome 32 representative genomes of all category were used for phylogenetic tree<br>development using MEGAX software (Kumar et al., 2018). The closest relative was found to be<br>SARS CoV. The glycoprotein region of SARS CoV and 2019-nCoV were aligned and visualized<br>using Multalin software (Corpet, 1988). The identified amino acid and nucleotide sequence were<br>aligned with whole viral genome database using BLASTp and BLASTn. The conservation of the<br>nucleotide and amino acid motifs in 28 clinical variants of 2019-nCoV genome were presented by<br>performing multiple sequence alignment using MEGAX software. The three dimensional structure<br>of 2019-nCoV glycoprotein was generated by using SWISS-MODEL online server (Biasini et al.,<br>2014) and the structure was marked and visualized by using PyMol (DeLano, 2002).<br>Results<br>Uncanny similarity of novel inserts in the 2019-nCoV spike protein to HIV-1 gp120 and<br>Gag<br>Our phylogentic tree of full-length coronaviruses suggests that 2019-nCoV is closely related to<br>SARS CoV [Fig1]. In addition, other recent studies have linked the 2019-nCoV to SARS CoV.<br>We therefore compared the spike glycoprotein sequences of the 2019-nCoV to that of the SARS<br>CoV (NCBI Accession number: AY390556.1). On careful examination of the sequence<br>alignment we found that the 2019- nCoV spike glycoprotein contains 4 insertions [Fig.2]. To<br>further investigate if these inserts are present in any other corona virus, we performed a multiple sequence alignment of the spike glycoprotein amino acid sequences of all available<br>coronaviruses (n=55) [refer Table S.File1] in NCBI refseq (ncbi.nlm.nih.gov) this includes one<br>sequence of 2019-nCoV[Fig.S1]. We found that these 4 insertions [inserts 1, 2, 3 and 4] are<br>unique to 2019-nCoV and are not present in other coronaviruses analyzed. Another group from<br>China had documented three insertions comparing fewer spike glycoprotein sequences of<br>coronaviruses . Another group from China had documented three insertions comparing fewer<br>spike glycoprotein sequences of coronaviruses (Zhou et al., 2020).<br>Figure 1: Maximum likelihood genealogy show the evolution of 2019- nCoV: The evolutionary history<br>was inferred by using the Maximum Likelihood method and JTT matrix-based model. The tree<br>with the highest log likelihood (12458.88) is shown. Initial tree(s) for the heuristic search were<br>obtained automatically by applying Neighbor-Join and BioNJ algorithms to a matrix of pairwise<br>distances estimated using a JTT model, and then selecting the topology with superior log likelihood value. This analysis involved 5 amino acid sequences. There were a total of 1387 positions in the<br>final dataset. Evolutionary analyses were conducted in MEGA X.<br>Figure 2: Multiple sequence alignment between spike proteins of 2019-nCoV and SARS. The<br>sequences of spike proteins of 2019-nCoV (Wuhan-HU-1, Accession NC_045512) and of SARS<br>CoV (GZ02, Accession AY390556) were aligned using MultiAlin software. The sites of difference<br>are highlighted in boxes.<br>We then analyzed all available full-length sequences (n=28) of 2019-nCoV in GISAID (Elbe &amp;<br>Buckland-Merrett, 2017) as on January 27, 2020 for the presence of these inserts. As most of these<br>sequences are not annotated, we compared the nucleotide sequences of the spike glycoprotein of<br>all available 2019-nCoV sequences using BLASTp. Interestingly, all the 4 insertions were<br>absolutely (100%) conserved in all the available 2019- nCoV sequences analyzed [Fig.S2, Fig.S3].<br>We then translated the aligned genome and found that these inserts are present in all Wuhan 2019-<br>nCoV viruses except the 2019-nCoV virus of Bat as a host [Fig.S4]. Intrigued by the 4 highly<br>conserved inserts unique to 2019-nCoV we wanted to understand their origin. For this purpose,<br>we used the 2019-nCoV local alignment with each insert as query against all virus genomes and<br>considered hits with 100% sequence coverage. Surprisingly, each of the four inserts aligned with<br>short segments of the Human immunodeficiency Virus-1 (HIV-1) proteins. The amino acid<br>positions of the inserts in 2019-nCoV and the corresponding residues in HIV-1 gp120 and HIV-1<br>Gag are shown in Table 1. The first 3 inserts (insert 1,2 and 3) aligned to short segments of amino<br>acid residues in HIV-1 gp120. The insert 4 aligned to HIV-1 Gag. The insert 1 (6 amino acid<br>residues) and insert 2 (6 amino acid residues) in the spike glycoprotein of 2019-nCoV are 100%<br>identical to the residues mapped to HIV-1 gp120. The insert 3 (12 amino acid residues) in 2019-<br>nCoV maps to HIV-1 gp120 with gaps [see Table 1]. The insert 4 (8 amino acid residues) maps to<br>HIV-1 Gag with gaps.<br>Although, the 4 inserts represent discontiguous short stretches of amino acids in spike glycoprotein<br>of 2019-nCoV, the fact that all three of them share amino acid identity or similarity with HIV-1<br>gp120 and HIV-1 Gag (among all annotated virus proteins) suggests that this is not a random<br>fortuitous finding. In other words, one may sporadically expect a fortuitous match for a stretch of<br>6-12 contiguous amino acid residues in an unrelated protein. However, it is unlikely that all 4<br>inserts in the 2019-nCoV spike glycoprotein fortuitously match with 2 key structural proteins of<br>an unrelated virus (HIV-1).<br>The amino acid residues of inserts 1, 2 and 3 of 2019-nCoV spike glycoprotein that mapped to<br>HIV-1 were a part of the V4, V5 and V1 domains respectively in gp120 [Table 1]. Since the 2019-<br>nCoV inserts mapped to variable regions of HIV-1, they were not ubiquitous in HIV-1 gp120, but<br>were limited to selected sequences of HIV-1 [ refer S.File1] primarily from Asia and Africa.<br>The HIV-1 Gag protein enables interaction of virus with negatively charged host surface<br>(Murakami, 2008) and a high positive charge on the Gag protein is a key feature for the host-virus<br>interaction. On analyzing the pI values for each of the 4 inserts in 2019-nCoV and the<br>corresponding stretches of amino acid residues from HIV-1 proteins we found that a) the pI values<br>were very similar for each pair analyzed b) most of these pI values were 10±2 [Refer Table 1] . Of<br>note, despite the gaps in inserts 3 and 4 the pI values were comparable. This uniformity in the pI<br>values for all the 4 inserts merits further investigation.<br>As none of these 4 inserts are present in any other coronavirus, the genomic region encoding these<br>inserts represent ideal candidates for designing primers that can distinguish 2019-nCoV from other<br>coronaviruses.<br>Table 1: Aligned sequences of 2019-nCoV and gp120 protein of HIV-1 with their positions<br>in primary sequence of protein. All the inserts have a high density of positively charged<br>residues. The deleted fragments in insert 3 and 4 increase the positive charge to surface area<br>ratio. *please see Supp. Table 1 for accession numbers<br>The novel inserts are part of the receptor binding site of 2019-nCoV<br>To get structural insights and to understand the role of these insertions in 2019-nCoV glycoprotein,<br>we modelled its structure based on available structure of SARS spike glycoprotein (PDB:<br>6ACD.1.A). The comparison of the modelled structure reveals that although inserts 1,2 and 3 are<br>at non-contiguous locations in the protein primary sequence, they fold to constitute the part of<br>glycoprotein binding site that recognizes the host receptor (Kirchdoerfer et al., 2016) (Figure 4).<br>The insert 1 corresponds to the NTD (N-terminal domain) and the inserts 2 and 3 correspond to<br>the CTD (C-terminal domain) of the S1 subunit in the 2019-nCoV spike glycoprotein. The insert<br>4 is at the junction of the SD1 (sub domain 1) and SD2 (sub domain 2) of the S1 subunit (Ou et<br>al., 2017). We speculate, that these insertions provide additional flexibility to the glycoprotein<br>binding site by forming a hydrophilic loop in the protein structure that may facilitate or enhance<br>virus-host interactions.<br>Figure 3. Modelled homo-trimer spike glycoprotein of 2019-nCoV virus. The inserts from HIV<br>envelop protein are shown with colored beads, present at the binding site of the protein.<br>Evolutionary Analysis of 2019-nCoV<br>It has been speculated that 2019-nCoV is a variant of Coronavirus derived from an animal source<br>which got transmitted to humans. Considering the change of specificity for host, we decided to<br>study the sequences of spike glycoprotein (S protein) of the virus. S proteins are surface proteins<br>that help the virus in host recognition and attachment. Thus, a change in these proteins can be<br>reflected as a change of host specificity of the virus. To know the alterations in S protein gene of<br>2019-nCoV and its consequences in structural re-arrangements we performed in-sillico analysis of<br>2019-nCoV with respect to all other viruses. A multiple sequence alignment between the S protein<br>amino acid sequences of 2019-nCoV, Bat-SARS-Like, SARS-GZ02 and MERS revealed that S<br>protein has evolved with closest significant diversity from the SARS-GZ02 (Figure 1).<br>Insertions in Spike protein region of 2019-nCoV<br>Since the S protein of 2019-nCoV shares closest ancestry with SARS GZ02, the sequence coding<br>for spike proteins of these two viruses were compared using MultiAlin software. We found four<br>new insertions in the protein of 2019-nCoV- “GTNGTKR” (IS1), “HKNNKS” (IS2), “GDSSSG”<br>(IS3) and “QTNSPRRA” (IS4) (Figure 2). To our surprise, these sequence insertions were not only<br>absent in S protein of SARS but were also not observed in any other member of the Coronaviridae<br>family (Supplementary figure). This is startling as it is quite unlikely for a virus to have acquired<br>such unique insertions naturally in a short duration of time.<br>Insertions share similarity to HIV<br>The insertions were observed to be present in all the genomic sequences of 2019-nCoV virus<br>available from the recent clinical isolates (Supplementary Figure 1). To know the source of these<br>insertions in 2019-nCoV a local alignment was done with BLASTp using these insertions as query<br>with all virus genome. Unexpectedly, all the insertions got aligned with Human immunodeficiency<br>Virus-1 (HIV-1). Further analysis revealed that aligned sequences of HIV-1 with 2019-nCoV were<br>derived from surface glycoprotein gp120 (amino acid sequence positions: 404-409, 462-467, 136-<br>150) and from Gag protein (366-384 amino acid) (Table 1). Gag protein of HIV is involved in host<br>membrane binding, packaging of the virus and for the formation of virus-like particles. Gp120<br>plays crucial role in recognizing the host cell by binding to the primary receptor CD4.This binding<br>induces structural rearrangements in GP120, creating a high affinity binding site for a chemokine<br>co-receptor like CXCR4 and/or CCR5.<br>Discussion<br>The current outbreak of 2019-nCoV warrants a thorough investigation and understanding of its<br>ability to infect human beings. Keeping in mind that there has been a clear change in the preference<br>of host from previous coronaviruses to this virus, we studied the change in spike protein between<br>2019-nCoV and other viruses. We found four new insertions in the S protein of 2019-nCoV when<br>compared to its nearest relative, SARS CoV. The genome sequence from the recent 28 clinical<br>isolates showed that the sequence coding for these insertions are conserved amongst all these<br>isolates. This indicates that these insertions have been preferably acquired by the 2019-nCoV,<br>providing it with additional survival and infectivity advantage. Delving deeper we found that these<br>insertions were similar to HIV-1. Our results highlight an astonishing relation between the gp120<br>and Gag protein of HIV, with 2019-nCoV spike glycoprotein. These proteins are critical for the<br>viruses to identify and latch on to their host cells and for viral assembly (Beniac et al., 2006).<br>Since surface proteins are responsible for host tropism, changes in these proteins imply a change<br>in host specificity of the virus. According to reports from China, there has been a gain of host<br>specificity in case 2019-nCoV as the virus was originally known to infect animals and not humans<br>but after the mutations, it has gained tropism to humans as well.<br>Moving ahead, 3D modelling of the protein structure displayed that these insertions are present at<br>the binding site of 2019-nCoV. Due to the presence of gp120 motifs in 2019-nCoV spike<br>glycoprotein at its binding domain, we propose that these motif insertions could have provided an<br>enhanced affinity towards host cell receptors. Further, this structural change might have also<br>increased the range of host cells that 2019-nCoV can infect. To the best of our knowledge, the<br>function of these motifs is still not clear in HIV and need to be explored. The exchange of genetic<br>material among the viruses is well known and such critical exchange highlights the risk and the<br>need to investigate the relations between seemingly unrelated virus families.<br>Conclusions<br>Our analysis of the spike glycoprotein of 2019-nCoV revealed several interesting findings: First,<br>we identified 4 unique inserts in the 2019-nCoV spike glycoprotein that are not present in any<br>other coronavirus reported till date. To our surprise, all the 4 inserts in the 2019-nCoV mapped to short segments of amino acids in the HIV-1 gp120 and Gag among all annotated virus proteins in<br>the NCBI database. This uncanny similarity of novel inserts in the 2019- nCoV spike protein to<br>HIV-1 gp120 and Gag is unlikely to be fortuitous. Further, 3D modelling suggests that atleast 3 of<br>the unique inserts which are non-contiguous in the primary protein sequence of the 2019-nCoV<br>spike glycoprotein converge to constitute the key components of the receptor binding site. Of note,<br>all the 4 inserts have pI values of around 10 that may facilitate virus-host interactions. Taken<br>together, our findings suggest unconventional evolution of 2019-nCoV that warrants further<br>investigation. Our work highlights novel evolutionary aspects of the 2019-nCoV and has<br>implications on the pathogenesis and diagnosis of this virus.<br>References<br>Beniac, D. R., Andonov, A., Grudeski, E., &amp; Booth, T. F. (2006). Architecture of the SARS coronavirus<br>prefusion spike. Nature Structural and Molecular Biology, 13(8), 751–752.<br><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nsmb1123">https://doi.org/10.1038/nsmb1123</a><br>Biasini, M., Bienert, S., Waterhouse, A., Arnold, K., Studer, G., Schmidt, T., Kiefer, F., Cassarino, T. G.,<br>Bertoni, M., Bordoli, L., &amp; Schwede, T. (2014). SWISS-MODEL: Modelling protein tertiary and<br>quaternary structure using evolutionary information. Nucleic Acids Research.<br><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gku340">https://doi.org/10.1093/nar/gku340</a><br>Bosch, B. J., van der Zee, R., de Haan, C. A. M., &amp; Rottier, P. J. M. (2003). The Coronavirus Spike Protein Is<br>a Class I Virus Fusion Protein: Structural and Functional Characterization of the Fusion Core<br>Complex. Journal of Virology, 77(16), 8801–8811. <a target="_blank" rel="noopener" href="https://doi.org/10.1128/jvi.77.16.8801-">https://doi.org/10.1128/jvi.77.16.8801-</a><br>8811.2003<br>Chan, J. F.-W., Kok, K.-H., Zhu, Z., Chu, H., To, K. K.-W., Yuan, S., &amp; Yuen, K.-Y. (2020). Genomic<br>characterization of the 2019 novel human-pathogenic coronavirus isolated from a patient with<br>atypical pneumonia after visiting Wuhan. Emerging Microbes &amp; Infections, 9(1), 221–236.<br><a target="_blank" rel="noopener" href="https://doi.org/10.1080/22221751.2020.1719902">https://doi.org/10.1080/22221751.2020.1719902</a><br>Chan, J. F. W., Lau, S. K. P., To, K. K. W., Cheng, V. C. C., Woo, P. C. Y., &amp; Yuen, K.-Y. (2015). Middle East<br>Respiratory Syndrome Coronavirus: Another Zoonotic Betacoronavirus Causing SARS-Like Disease.<br><a target="_blank" rel="noopener" href="https://doi.org/10.1128/CMR.00102-14">https://doi.org/10.1128/CMR.00102-14</a><br>Chan, J., To, K., Tse, H., Jin, D., microbiology, K. Y.-T. in, &amp; 2013, undefined. (n.d.). Interspecies<br>transmission and emergence of novel viruses: lessons from bats and birds. Elsevier.<br>Corpet, F. (1988). Multiple sequence alignment with hierarchical clustering. Nucleic Acids Research.<br><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/16.22.10881">https://doi.org/10.1093/nar/16.22.10881</a><br>DeLano, W. L. (2002). The PyMOL Molecular Graphics System, Version 1.1. Schr{ö}dinger LLC.<br><a target="_blank" rel="noopener" href="https://doi.org/10.1038/hr.2014.17">https://doi.org/10.1038/hr.2014.17</a><br>Du, L., Zhao, G., Kou, Z., Ma, C., Sun, S., Poon, V. K. M., Lu, L., Wang, L., Debnath, A. K., Zheng, B.-J., Zhou,<br>Y., &amp; Jiang, S. (2013). Identification of a Receptor-Binding Domain in the S Protein of the Novel<br>Human Coronavirus Middle East Respiratory Syndrome Coronavirus as an Essential Target for<br>Vaccine Development. Journal of Virology, 87(17), 9939–9942. <a target="_blank" rel="noopener" href="https://doi.org/10.1128/jvi.01048-">https://doi.org/10.1128/jvi.01048-</a><br>13<br>Edgar, R. C. (2004). MUSCLE: Multiple sequence alignment with high accuracy and high throughput.<br>Nucleic Acids Research. <a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkh340">https://doi.org/10.1093/nar/gkh340</a><br>Elbe, S., &amp; Buckland-Merrett, G. (2017). Data, disease and diplomacy: GISAID’s innovative contribution<br>to global health. Global Challenges. <a target="_blank" rel="noopener" href="https://doi.org/10.1002/gch2.1018">https://doi.org/10.1002/gch2.1018</a><br>Kirchdoerfer, R. N., Cottrell, C. A., Wang, N., Pallesen, J., Yassine, H. M., Turner, H. L., Corbett, K. S.,<br>Graham, B. S., McLellan, J. S., &amp; Ward, A. B. (2016). Pre-fusion structure of a human coronavirus<br>spike protein. Nature. <a target="_blank" rel="noopener" href="https://doi.org/10.1038/nature17200">https://doi.org/10.1038/nature17200</a><br>Kumar, S., Stecher, G., Li, M., Knyaz, C., &amp; Tamura, K. (2018). MEGA X: Molecular evolutionary genetics<br>analysis across computing platforms. Molecular Biology and Evolution.<br><a target="_blank" rel="noopener" href="https://doi.org/10.1093/molbev/msy096">https://doi.org/10.1093/molbev/msy096</a><br>Li, F. (2016). Structure, Function, and Evolution of Coronavirus Spike Proteins. Annual Review of<br>Virology, 3(1), 237–261. <a target="_blank" rel="noopener" href="https://doi.org/10.1146/annurev-virology-110615-042301">https://doi.org/10.1146/annurev-virology-110615-042301</a><br>Murakami, T. (2008). Roles of the interactions between Env and Gag proteins in the HIV-1 replication<br>cycle. Microbiology and Immunology, 52(5), 287–295. <a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1348-">https://doi.org/10.1111/j.1348-</a><br>0421.2008.00008.x<br>Ou, X., Guan, H., Qin, B., Mu, Z., Wojdyla, J. A., Wang, M., Dominguez, S. R., Qian, Z., &amp; Cui, S. (2017).<br>Crystal structure of the receptor binding domain of the spike glycoprotein of human<br>betacoronavirus HKU1. Nature Communications. <a target="_blank" rel="noopener" href="https://doi.org/10.1038/ncomms15216">https://doi.org/10.1038/ncomms15216</a><br>Snijder, E. J., van der Meer, Y., Zevenhoven-Dobbe, J., Onderwater, J. J. M., van der Meulen, J., Koerten,<br>H. K., &amp; Mommaas, A. M. (2006). Ultrastructure and origin of membrane vesicles associated with<br>the severe acute respiratory syndrome coronavirus replication complex. Journal of Virology,<br>80(12), 5927–5940. <a target="_blank" rel="noopener" href="https://doi.org/10.1128/JVI.02501-05">https://doi.org/10.1128/JVI.02501-05</a><br>Zhou, P., Yang, X.-L., Wang, X.-G., Hu, B., Zhang, L., Zhang, W., Si, H.-R., Zhu, Y., Li, B., Huang, C.-L., Chen,<br>H.-D., Chen, J., Luo, Y., Guo, H., Jiang, R.-D., Liu, M.-Q., Chen, Y., Shen, X.-R., Wang, X., … Shi, Z.-L.<br>(2020). Discovery of a novel coronavirus associated with the recent pneumonia outbreak in<br>humans and its potential bat origin. BioRxiv. <a target="_blank" rel="noopener" href="https://doi.org/10.1101/2020.01.22.914952">https://doi.org/10.1101/2020.01.22.914952</a><br>Zhu, N., Zhang, D., Wang, W., Li, X., Yang, B., Song, J., Zhao, X., Huang, B., Shi, W., Lu, R., Niu, P., Zhan, F.,<br>Ma, X., Wang, D., Xu, W., Wu, G., Gao, G. F., &amp; Tan, W. (2020). A Novel Coronavirus from Patients<br>with Pneumonia in China, 2019. New England Journal of Medicine, NEJMoa2001017.<br><a target="_blank" rel="noopener" href="https://doi.org/10.1056/NEJMoa2001017">https://doi.org/10.1056/NEJMoa2001017</a><br>Fig.S1 Multiple sequence alignment of glycoprotein of coronaviridae family, representing all the<br>four inserts.<br>Fig.S2: All four inserts are present in the aligned 28 Wuhan 2019-nCoV virus genomes obtained<br>from GISAID. The gap in the Bat-SARS Like CoV in the last row shows that insert 1 and 4 is very<br>unique to Wuhan 2019-nCoV.<br>Fig.S3 Phylogenetic tree of 28 clinical isolates genome of 2019-nCoV including one from bat as a host.<br>Supplementary Fig 4. Genome alingment of Coronaviridae family. Highlighted black sequences are the<br>inserts represented here. </p>
</blockquote>

    </div>

    
    
    
        

<div>
<ul class="post-copyright">
  <li class="post-copyright-author">
    <strong>本文作者： </strong>Dana Shaw
  </li>
  <li class="post-copyright-link">
    <strong>本文链接：</strong>
    <a href="https://dana5haw.com/posts/ZH-CN/2019-nCov.html" title="2019-nCov病毒">https://dana5haw.com/posts/ZH-CN/2019-nCov.html</a>
  </li>
  <li class="post-copyright-license">
    <strong>版权声明： </strong>本博客所有文章除特别声明外，均采用 <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/deed.zh" rel="noopener" target="_blank"><i class="fa fa-fw fa-creative-commons"></i>BY-NC-SA</a> 许可协议。转载请注明出处！
  </li>

  <div class="wechat_OA">
	<span>欢迎关注我的公众号(Welcome to follow me on Wechat)</span>
	<br>
    <!-- 这里添加你的二维码图片 -->
	<img src ="https://v2ray.dana5haw.com:8008/images/2021/08/03/qrcode_for_gh_9b7280eb1edc_258.jpg">
</div>

</ul>
</div>


      <div>   
         
        <div>     
             
    <div style="text-align:center;color: #ccc;font-size:14px;">
        -------------本文结束<i class="fa fa-paw"></i>感谢您的阅读-------------
    </div>     
     
</div>   
     
      </div>

      <footer class="post-footer">
          
          <div class="post-tags">
              <a href="/tags/%E7%BF%BB%E8%AF%91/" rel="tag"><i class="fa fa-tag"></i> 翻译</a>
              <a href="/tags/2019-nCov%E7%97%85%E6%AF%92/" rel="tag"><i class="fa fa-tag"></i> 2019-nCov病毒</a>
          </div>

        


        
    <div class="post-nav">
      <div class="post-nav-item">
    <a href="/posts/ZH-CN/How_To_Type.html" rel="prev" title="输入法之怎么输入特殊符号">
      <i class="fa fa-chevron-left"></i> 输入法之怎么输入特殊符号
    </a></div>
      <div class="post-nav-item">
    <a href="/posts/ZH-CN/Reg_Exclude.html" rel="next" title="正则表达式之排除匹配">
      正则表达式之排除匹配 <i class="fa fa-chevron-right"></i>
    </a></div>
    </div>
      </footer>
    
  </article>
  
  
  

  </div>


          </div>
          
    <div class="comments" id="gitalk-container"></div>

<script>
  window.addEventListener('tabs:register', () => {
    let activeClass = CONFIG.comments.activeClass;
    if (CONFIG.comments.storage) {
      activeClass = localStorage.getItem('comments_active') || activeClass;
    }
    if (activeClass) {
      let activeTab = document.querySelector(`a[href="#comment-${activeClass}"]`);
      if (activeTab) {
        activeTab.click();
      }
    }
  });
  if (CONFIG.comments.storage) {
    window.addEventListener('tabs:click', event => {
      if (!event.target.matches('.tabs-comment .tab-content .tab-pane')) return;
      let commentClass = event.target.classList[1];
      localStorage.setItem('comments_active', commentClass);
    });
  }
</script>

        </div>
          
  
  <div class="toggle sidebar-toggle">
    <span class="toggle-line toggle-line-first"></span>
    <span class="toggle-line toggle-line-middle"></span>
    <span class="toggle-line toggle-line-last"></span>
  </div>

  <aside class="sidebar">
    <div class="sidebar-inner">

      <ul class="sidebar-nav motion-element">
        <li class="sidebar-nav-toc">
          文章目录
        </li>
        <li class="sidebar-nav-overview">
          站点概览
        </li>
      </ul>

      <!--noindex-->
      <div class="post-toc-wrap sidebar-panel">
          <div class="post-toc motion-element"><ol class="nav"><li class="nav-item nav-level-2"><a class="nav-link" href="#%E5%89%8D%E8%A8%80"><span class="nav-number">1.</span> <span class="nav-text">前言</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#%E6%AD%A3%E6%96%87"><span class="nav-number">2.</span> <span class="nav-text">正文</span></a></li></ol></div>
      </div>
        <div id="">
	  <canvas id="canvas" style="width:60%;">
	    </div>
	      <script type="text/javascript" src="/js/canvas-dance-time.js"></script>
      <!--/noindex-->


      <div class="site-overview-wrap sidebar-panel">
        <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
    <img class="site-author-image" itemprop="image" alt="Dana Shaw"
      src="/./images/photo.jpg">
  <p class="site-author-name" itemprop="name">Dana Shaw</p>
  <div class="site-description" itemprop="description">连尘埃也想收藏</div>
</div>
<div class="site-state-wrap motion-element">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
          <a href="/archives/">
        
          <span class="site-state-item-count">242</span>
          <span class="site-state-item-name">日志</span>
        </a>
      </div>
      <div class="site-state-item site-state-categories">
            <a href="/categories/">
          
        <span class="site-state-item-count">85</span>
        <span class="site-state-item-name">分类</span></a>
      </div>
      <div class="site-state-item site-state-tags">
            <a href="/tags/">
          
        <span class="site-state-item-count">399</span>
        <span class="site-state-item-name">标签</span></a>
      </div>
  </nav>
</div>
  <div class="links-of-author motion-element">
      <span class="links-of-author-item">
        <a href="https://github.com/DanaShaw" title="GitHub → https:&#x2F;&#x2F;github.com&#x2F;DanaShaw" rel="noopener" target="_blank"><i class="fa fa-fw fa-github"></i>GitHub</a>
      </span>
      <span class="links-of-author-item">
        <a href="mailto:danaxkj@gmail.com" title="E-Mail → mailto:danaxkj@gmail.com" rel="noopener" target="_blank"><i class="fa fa-fw fa-envelope"></i>E-Mail</a>
      </span>
      <span class="links-of-author-item">
        <a href="https://weibo.com/u/5742631593" title="Weibo → https:&#x2F;&#x2F;weibo.com&#x2F;u&#x2F;5742631593" rel="noopener" target="_blank"><i class="fa fa-fw fa-weibo"></i>Weibo</a>
      </span>
      <span class="links-of-author-item">
        <a href="/atom.xml" title="RSS → &#x2F;atom.xml"><i class="fa fa-fw fa-rss"></i>RSS</a>
      </span>
  </div>


  <div class="links-of-blogroll motion-element">
    <div class="links-of-blogroll-title">
      <i class="fa fa-fw fa-link"></i>
      友情链接
    </div>
    <ul class="links-of-blogroll-list">
        <li class="links-of-blogroll-item">
          <a href="https://whale3070.github.io/" title="https:&#x2F;&#x2F;whale3070.github.io&#x2F;" rel="noopener" target="_blank">windows提权学习</a>
        </li>
    </ul>
  </div>

      </div>

    </div>
  </aside>
  <div id="sidebar-dimmer"></div>


      </div>
    </main>

    <footer class="footer">
      <div class="footer-inner">
        

<script async src="https://dn-lbstatics.qbox.me/busuanzi/2.3/busuanzi.pure.mini.js"></script>

<span id="timeDate">载入天数...</span>
<span id="times">载入时分秒...</span>
<script>
    var now = new Date();
        function createtime() {
	        var grt= new Date("01/07/2020 21:49:45");//此处修改你的建站时间或者网站上线时间
		        now.setTime(now.getTime()+250);
			        days = (now - grt ) / 1000 / 60 / 60 / 24; dnum = Math.floor(days);
				        hours = (now - grt ) / 1000 / 60 / 60 - (24 * dnum); hnum = Math.floor(hours);
					        if(String(hnum).length ==1 ){hnum = "0" + hnum;} minutes = (now - grt ) / 1000 /60 - (24 * 60 * dnum) - (60 * hnum);
						        mnum = Math.floor(minutes); if(String(mnum).length ==1 ){mnum = "0" + mnum;}
							        seconds = (now - grt ) / 1000 - (24 * 60 * 60 * dnum) - (60 * 60 * hnum) - (60 * mnum);
								        snum = Math.round(seconds); if(String(snum).length ==1 ){snum = "0" + snum;}
									        document.getElementById("timeDate").innerHTML = "本站已安全运行 "+dnum+" 天 ";
										        document.getElementById("times").innerHTML = hnum + " 小时 " + mnum + " 分 " + snum + " 秒";
											    }
											    setInterval("createtime()",250);
											    </script>

<div class="copyright">
  
  &copy; 2020 – 
  <span itemprop="copyrightYear">2021</span>
  <span class="with-love">
    <i class="fa fa-user"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">Dana Shaw</span>
    <span class="post-meta-divider">|</span>
    <span class="post-meta-item-icon">
      <i class="fa fa-area-chart"></i>
    </span>
    <span title="站点总字数">526k</span>
    <span class="post-meta-divider">|</span>
    <span class="post-meta-item-icon">
      <i class="fa fa-coffee"></i>
    </span>
    <span title="站点阅读时长">7:58</span>
</div>


  <script src='https://unpkg.com/mermaid@/dist/mermaid.min.js'></script>
  <script>
    if (window.mermaid) {
      mermaid.initialize({theme: 'default'});
    }
  </script>

        
<div class="busuanzi-count">
  <script async src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
    <span class="post-meta-item" id="busuanzi_container_site_uv" style="display: none;">
      <span class="post-meta-item-icon">
        <i class="fa fa-user"></i>
      </span>
      <span class="site-uv" title="总访客量">
        <span id="busuanzi_value_site_uv"></span>
      </span>
    </span>
    <span class="post-meta-divider">|</span>
    <span class="post-meta-item" id="busuanzi_container_site_pv" style="display: none;">
      <span class="post-meta-item-icon">
        <i class="fa fa-eye"></i>
      </span>
      <span class="site-pv" title="总访问量">
        <span id="busuanzi_value_site_pv"></span>
      </span>
    </span>
</div>








      </div>
    </footer>
  </div>

  
  
  <script color='0,0,255' opacity='0.5' zIndex='-1' count='99' src="/lib/canvas-nest/canvas-nest.min.js"></script>
  <script src="/lib/anime.min.js"></script>
  <script src="//cdn.jsdelivr.net/npm/jquery@3/dist/jquery.min.js"></script>
  <script src="//cdn.jsdelivr.net/gh/fancyapps/fancybox@3/dist/jquery.fancybox.min.js"></script>
  <script src="/lib/velocity/velocity.min.js"></script>
  <script src="/lib/velocity/velocity.ui.min.js"></script>

<script src="/js/utils.js"></script>

<script src="/js/motion.js"></script>


<script src="/js/schemes/pisces.js"></script>


<script src="/js/next-boot.js"></script>

<script src="/js/bookmark.js"></script>




  




  
<script src="/js/local-search.js"></script>











<script>
if (document.querySelectorAll('pre.mermaid').length) {
  NexT.utils.getScript('//cdn.jsdelivr.net/npm/mermaid@8/dist/mermaid.min.js', () => {
    mermaid.initialize({
      theme: 'default',
      logLevel: 3,
      flowchart: { curve: 'linear' },
      gantt: { axisFormat: '%m/%d/%Y' },
      sequence: { actorMargin: 50 }
    });
  }, window.mermaid);
}
</script>


  

  

<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/gitalk@1/dist/gitalk.min.css">

<script>
NexT.utils.loadComments(document.querySelector('#gitalk-container'), () => {
  NexT.utils.getScript('//cdn.jsdelivr.net/npm/gitalk@1/dist/gitalk.min.js', () => {
    var gitalk = new Gitalk({
      clientID: '2dc357f83ef2282cb7cb',
      clientSecret: '64ad793540c649b1f67bcbd2fc4c648b631176c5',
      repo: 'DanaShaw.github.io',
      owner: 'DanaShaw',
      admin: ['DanaShaw'],
      id: '3cc2b49eaae2c3cd2a39c215f5a6013d',
        language: 'zh-CN',
      distractionFreeMode: true
    });
    gitalk.render('gitalk-container');
  }, window.Gitalk);
});
</script>

<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=ca-pub-3938956891324222"
     crossorigin="anonymous"></script>
<!-- sidebar -->
<ins class="adsbygoogle"
     style="display:block"
     data-ad-client="ca-pub-3938956891324222"
     data-ad-slot="6163650450"
     data-ad-format="auto"
     data-full-width-responsive="true"></ins>
<script>
     (adsbygoogle = window.adsbygoogle || []).push({});
</script>
  
  <script type="text/javascript" src="//cdn.bootcss.com/canvas-nest.js/1.0.0/canvas-nest.min.js"></script>
  
  
   <canvas class="fireworks" style="position: fixed;left: 0;top: 0;z-index: 1; pointer-events: none;" ></canvas> 
   <script type="text/javascript" src="//cdn.bootcss.com/animejs/2.2.0/anime.min.js"></script> 
   <script type="text/javascript" src="/js/fireworks.js"></script>

</body>
</html>
